test/test_dataloader.py

```diff
@@ -205,9 +205,12 @@ class SleepDataset(Dataset):
     def __init__(self, size, sleep_sec):
         self.size = size
         self.sleep_sec = sleep_sec
+        self.sleeped = False
 
     def __getitem__(self, idx):
-        time.sleep(self.sleep_sec)
+        if not self.sleeped:
+            time.sleep(self.sleep_sec)
+            self.sleeped = True
```

Comment:
Why do we change it to a one time sleep now?

Reply:
To speed up the test. The workers won't quit until they finish the second batch (or all existing batches before `done_event` is used).